CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## 0.6.1
+
```

Comment:
Should we mention the pre-commit change here, considering that pre-commit package does not have its own changelog

Reply:
I considered this. I'll add a note

Reply:
We could update the release notes once it's "auto" released?

Reply:
I added a note about the pre-commit issue; how's that look?